fs/task_tracker_itis_2023
# Added HW7

### Summary

A brief description of the pull request.

### How it works

Screenshots/screencasts of the pull request introduced functionality.

### Test plan

List of steps to manually test introduced functionality:

- Log in
- Go to http://localhost:3000
- Check user role in database that be admin
### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, services, scripts, etc.

### Checklist:
